api/approx_count_distinct.md

```diff
@@ -0,0 +1,59 @@
+---
+api_name: approx_count_distinct()
+excerpt: Aggregate data into a hyperloglog for approximate counting without having to specify the number of buckets.
+license: community
+toolkit: true
+topic: hyperfunctions
+tags: [hyperfunctions, approximate count distinct, distinct count, hyperloglog]
+api_category: hyperfunction
+toolkit-experimental: false
+hyperfunction_toolkit: true
+hyperfunction_family: 'approximate count distinct'
+hyperfunction_subfamily: hyperloglog
+hyperfunction_type: aggregate
+---
+
+# approx_count_distinct()  <tag type="toolkit">Toolkit</tag>
+The `approx_count_distinct` function constructs and returns a hyperloglog with at least
+32,768 buckets over the given values.
```

Comment:
I believe it's actually at most 32,768 buckets.  That being said, I would try to avoid talking about buckets here as this should be more accessible to users.  Maybe "a hyperloglog with a default size appropriate for the majority of use cases"?